test/integrationtests.jl

```diff
@@ -359,6 +339,7 @@ end
                 b                         \|    1      1  \s*\d*.\ds
             src/bar_tests.jl              \|    4      4  \s*
               bar                         \|    4      4  \s*\d*.\ds
+                bar values                \|    2      2  \s*\d*.\ds
             src/foo_test.jl               \|    2      2  \s*
               foo                         \|    2      2  \s*\d*.\ds
         """
```

Comment:
Can you add a test with verbose_results=false and test that the response is just the expected two lines?